enterprise-suite/console-api/prometheus.yml

```diff
@@ -88,6 +88,15 @@ scrape_configs:
         regex: (.*)-[0-9]{13}-driver;
         replacement: ${1}-driver
         target_label: es_workload
+    # flink workload — of the form 'taxi-ride-fare-seglo-processor-dde77c7d-jm' or 'taxi-ride-fare-seglo-processor-dde77c7d-tm'
+      - source_labels: [kubernetes_pod_name, es_workload]
+        regex: (.*)-[a-z,0-9]+-tm;
+        replacement: ${1}-tm
+        target_label: es_workload
+      - source_labels: [kubernetes_pod_name, es_workload]
+        regex: (.*)-[a-z,0-9]+-jm;
```

Comment:
I think you can combine this with the previous one. Use regex: `(.*)-[a-z,0-9]+-(jm|tm);`